### ISSUE TITLE
Configuration example text on website is illegible

### ISSUE THREAD

**Issue by martinpub** [daisy/ace#404]
I noticed that on https://daisy.github.io/ace/docs/config/#defaults, the text is illegible due to insufficient contrast.

It seems the wrapping `<pre>` carries some extra styling that is probably not relevant.

**Comment by danielweck** [comment]
Indeed! I see this in both light and dark modes (MacOS). I wonder why the background is `none` in the CSS:

https://github.com/daisy/ace/blob/d56a8e1c927c49ee450ceaaa670efdb21dd2fb5f/website/themes/daisy/static/css/prism.css#L10-L11
